### PR TITLE
Add more variations of simialr

### PR DESCRIPTION
**Pull request by Akuli**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10532,6 +10532,7 @@ disscussing->discussing
 disscussion->discussion
 disscussions->discussions
 disshearteningly->dishearteningly
+dissimialrity->dissimilarity
 dissimiliarity->dissimilarity
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:10536]: A few more variations
```suggestion
dissimialr->dissimilar
dissimialrity->dissimilarity
dissimiliar->dissimilar
dissimiliarity->dissimilarity
```

**Review by peternewman — CHANGES_REQUESTED** [review]
More variations please

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Akuli) [reply codespell_lib/data/dictionary.txt:10536]: I took all lines containing `->similar` and prefixed them with `dis`.

**Review by peternewman — APPROVED** [review]
LGTM thanks.

**Pull request merged**